tests/dummy/app/templates/docs/usage.md

```diff
@@ -0,0 +1,37 @@
+# Usage
+
+## The dfp-ad Component
+
+```javascript
+  {{dfp-ad
+    slotClassNames="l-constrained aligncenter leaderboard"
+    slot="leaderboard/wqxr_leaderboard_demo"
+    target="leaderboard_ad_home"
+    mapping=(array (array 0 (array 300 50)) (array 758 (array 728 90)) (array 1203 (array 970 415)))
+    sizes=(array (array 970 415) (array 728 90) (array 300 50))
+    slotRenderEndedAction=(action 'handleSlotRendered')
+    clearOnEmptyRefresh=false
+  }}
+```
+
+Things you might want to know when using this addon:
+
+* For an ad to display, both the full slot name and the ad size have to match something being served out of DFP
+
+* the `slot` name above is `6483581/leaderboard/wqxr_leaderboard_demo`
+
+* The `mapping` parameter above says:
+  * for screen sizes between 0px wide and 757px wide, use ads from the `300 x 50` slot
+  * for screen sizes between 757px wide and 1203, use ads from the `728 x 90` slot
```

Comment:
I believe this should say `for screen sizes between 758px wide and 1202, ...`, i.e. the range for these is inclusive of the minimum value, but exclusive of the value in the range above, or put another way:
- `>= 0 && < 758`
- `>= 758 && < 1203`
- `>= 1203`

That's my understanding. Please correct me if I'm wrong!

Reply:
I just copy-pasted this part from the original readme, i'll double check how it works though.